libyul/AsmAnalysis.cpp

```diff
@@ -388,7 +378,6 @@ bool AsmAnalyzer::operator()(Switch const& _switch)
 	if (!expectExpression(*_switch.expression))
 		success = false;
 
-	if (m_dialect.flavour == AsmFlavour::Yul)
 	{
```

Comment:
Can you add an another commit here which removes this block and fixes indentation?

Reply:
Updated.